src/portal-components/injectPropsIntoConfig.ts

```diff
@@ -17,6 +17,7 @@ const injectPropsIntoConfig = (
   el: RowSynapseConfig,
   props: any,
 ): any => {
+  // If isValueSourceEntity, then 
```

Comment:
Is this relevant? Not sure what this means/what the alternative is

Reply:
Removed